editor/utils/mobile/index.js

```diff
@@ -1,17 +1,24 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
 /**
  * Internal dependencies
  */
 import { isMobile } from '../../store/selectors';
 import { toggleSidebar } from '../../store/actions';
 
 /**
- * Disables isSidebarOpened on rehydrate payload if the user is on a mobile screen size.
+ * Disables mobile sidebar if it is present on the payload.
  *
  * @param  {Object} payload rehydrate payload
- * @return {Object}         rehydrate payload with isSidebarOpened disabled if on mobile
+ * @return {Object}         rehydrate payload with mobile sidebar disabled
  */
-export const disableIsSidebarOpenedOnMobile = ( payload ) => (
-	payload.isSidebarOpenedMobile ? { ...payload, isSidebarOpenedMobile: false } : payload
+export const disableMobileSidebar = ( payload ) => (
+	get( payload, 'sidebars.mobile' ) ?
+		{ ...payload, ...{ sidebars: { ...payload.sidebars, mobile: false } } } :
```

Comment:
Apologies upfront for my incompetence, but is the second spread operator necessary in this case? isn't this the same as `{ ...payload, sidebars: { ...payload.sidebars, mobile: false } }`?

Reply:
Hi @chriskmnds, you are totally right 👍 your code is equivalent and simpler.